src/lib/generic-views.tsx

```diff
@@ -31,6 +31,7 @@ export class PreRenderedView extends ShapeView {
             return undefined;
         }
         const node = virtualize(model.code);
+        if (node === null) throw new Error('node must be single Element');
```

Comment:
Modified the virtualize function to return the error of DOMParser as a vnode instead of throwing an exception. Also modified the view to return undefined instead of throwing an exception.